### PR TITLE
Better unicorn error reporting.

### PR DESCRIPTION
**Pull request by 1whatleytay**
Tells us specific error names plus register values.

velo was talking about tracking down which module the error might've happened in
that might be cool to add

### CODE REVIEW

File: vita3k/cpu/src/cpu.cpp [hunk 1]
```diff
@@ -219,10 +238,11 @@ int run(CPUState &state, bool callback) {
     err = uc_emu_start(state.uc.get(), pc, state.entry_point & 0xfffffffe, 0, 0);
 
     if (err != UC_ERR_OK) {
-        std::uint32_t error_pc = read_pc(state);
-        uint32_t lr = read_lr(state);
-        LOG_CRITICAL("Unicorn error {} at: start PC: {} error PC {} LR: {}",
-            log_hex(err), log_hex(pc), log_hex(error_pc), log_hex(lr));
+//        std::uint32_t error_pc = read_pc(state);
```
Comment (VelocityRa) [inline vita3k/cpu/src/cpu.cpp]: you can delete these no?

Comment (1whatleytay) [reply vita3k/cpu/src/cpu.cpp]: Right, amended.

**Comment by 1whatleytay** [comment]
Will merge after travis :D

**Comment by VelocityRa** [comment]
No need to wait for CI if you're sure it builds btw.

**Pull request merged**